Description: Ignore emoji pack in super-large emoji rendering
 It is necessary to build against the unpatched rlottie lib.
 .
 Although latest versions of the library already support theme overriding, we
 have to patch the rlottie call here due to a stale version still available on
 IA-64 platform.
Bug-Samsung: https://github.com/Samsung/rlottie/pull/252
Forwarded: not-needed
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Tue, 25 Aug 2020 23:14:30 +0300

--- a/Telegram/lib_lottie/lottie/lottie_animation.cpp
+++ b/Telegram/lib_lottie/lottie/lottie_animation.cpp
@@ -146,11 +146,7 @@ std::unique_ptr<rlottie::Animation> CreateFromContent(
 			? replacements->replacements
 			: std::vector<std::pair<std::uint32_t, std::uint32_t>>()));
 #else
-	auto result = rlottie::Animation::loadFromData(
-		string,
-		std::string(),
-		std::string(),
-		false);
+	auto result = rlottie::Animation::loadFromData(string, "");
 #endif
 	return result;
 }
